azureadgraph_connector.py

```diff
@@ -333,6 +333,7 @@ def _get_error_message_from_exception(self, e):
                 elif len(e.args) == 1:
                     error_msg = e.args[0]
         except Exception:
+            self.error_print("Error occured while getting message from exeption.")
```

Comment:
Seeing that other `error_print` above includes the the Exception object as well, should it be included here as well?

Reply:
Yes, Added exception object as added in other error_print.